index.js

```diff
@@ -17,100 +17,58 @@ let maxPage = 1;
 let page = 1;
 let searchQuery = "";
 
-async function fetchCharacters(searchQueryObject) {
```

Comment:
From the instructions, I believe we shouldn't pass any parameter here. We can use the state variable `searchQuery` directly here.